tasks/config.yml

```diff
@@ -72,6 +72,7 @@
   command: bbb-conf --secret
   changed_when: false
   register: result
+  check_mode: no
```

Comment:
This change seems unrelated to mp4? Or what does this change/fix?

Reply:
Otherwise the execution in check-mode will fail. But yes, I committed that by accident and this should probably be in a separate PR.